service_capacity_modeling/models/org/netflix/aurora.py

```diff
@@ -161,7 +180,7 @@ def _estimate_aurora_regional(
     extra_model_arguments: Dict[str, Any],
 ) -> Optional[CapacityPlan]:
     instance_family = instance.family
-    if instance_family not in ("x2g", "r6g", "r6i", "r5", "t4g"):  # TODO: split db instance and ec2 instance
+    if instance_family not in ("db.x2g", "db.r6g", "db.r6i", "db.r5", "db.t4g"):
```

Comment:
could this be `not instance_family.startswith('db.')`?

Reply:
There are only some of the instance families are supported in Aurora. For example, db.m5 is supported by RDS but not Aurora.